peps/pep-0772.rst

```diff
@@ -0,0 +1,463 @@
+
+This PEP currently mirrors the Python Steering Council's limit, that at most
+two individuals related to a single organisation can be on the council.
+
+Limiting it to one is workable; although it hasn't come up in the SC, people do
+move around, and we wouldn't want good candidates to either make employment
+decisions based on PC membership, or have to resign based on an employment
+change.  Limiting it to max 2, plus votes of no confidence is probably
+sufficient safety from any undue employer influence.
+
+---------------------------------------------------------------------------
+Establishing specific processes for Packaging Council and PyPA relationship
+---------------------------------------------------------------------------
+
+As noted in the abstract, the focus of this PEP is on providing a
+minimal-but-solid foundation for further governance decisions. The specifics of
+this relationship would be figured out by the inaugural council.
+
+=======================================
+Appendix: Approval process for this PEP
+=======================================
+
+This PEP would likely require an atypical process for approval given that it
+requires changes to PyPA's governance (which involves a PyPA-committers vote)
+and it requires the Python Steering Council to change their delegations.
+
+To that end, the process for approval for this PEP will be:
+
+* Submit this PEP for a vote on the pypa-committers mailing list, in accordance
+  with the process outlined in :pep:`609`.
+* Submit this PEP for the Python Steering Council's comments and approval.
+* Reconcile any outstanding variances in text and repeat, if necessary.
+
+=================================================
+Appendix: Operational suggestions for the council
+=================================================
+
+This section is based on what the PEP's authors view as things that would be
+beneficial for the Packaging Council to establish operational processes for.
+These are non-binding yet strongly encouraged.
+
+The PSF will designate a staff person to be the Packaging Council's official
+liaison who will regularly attend meetings, since it is expected that the
+Packaging Council will meet on a regular basis (twice a month).
+
+* Coordinate with the Steering Council on PEPs that need input from both
+  groups.
+* Coordinate with PyPA on their ongoing work to support individual projects.
+* Delegate to domain experts or working groups in the Packaging Community, for
+  initiatives/PEPs with a niche focus (analogous to how the Steering Council
+  sends certain PEPs to the C API working group).
+* Scope out work that might best be done by hiring someone and then work with
+  PSF to establish outcomes and a reasonable budget.
+* The Packaging Council (similar to the Steering Council) is encouraged to
+  communicate with and when necessary seek advice from the PSF's Conduct
+  Working Group.
+* Find ways to regularly synchronise with the Steering Council.
```

Comment:
Perhaps initially a quarterly joint meeting.

Reply:
I think the SC and PC could work out the cadence, much like the SC does with the DiR syncs.

Reply:
Maybe recommend an initial cadence, subject to change by SC and PC,  so the expectation isn't weekly or monthly.

Reply:
How about "Regularly synchronize with the Steering Council on a mutually agreed cadence, with a recommended frequency of no less than once per quarter."

Reply:
@warsaw Perfect! TYVM.